src/items/homeland-item/homelandSheet.ts

```diff
@@ -72,76 +72,4 @@ export class HomelandSheet extends RqgItemSheet<
 
     form.addEventListener("drop", this._onDrop.bind(this));
   }
-
```

Comment:
@wake42  none of this special handling was needed, so I removed the entire method and it just gets handled at the ItemSheet level.